docs/howtos/rate-limiting-tutorial.md

```diff
@@ -1,16 +1,18 @@
 # Rate Limiting
 
-The Ambassador Edge Stack can validate incoming requests before routing them to a backing service. In this tutorial, we'll configure the Ambassador Edge Stack to use a simple third party rate limit service. If you don't want to implement your own rate limiting service, the Ambassador Edge Stack integrates a [powerful, flexible rate limiting service](../../topics/using/rate-limits/).
+IMPORTANT: This guide applies to the Ambassador API Gateway version of Ambassador, use of this guide on the Ambassador Edge Stack is not fully supported.  Use the existing [RateLimitService](../../topics/using/rate-limits/) instead.
```

Comment:
I would use the same phrasing here as (what I suggested) for the basic auth guide, i.e., something more like:

IMPORTANT: This guide applies to the Ambassador API Gateway. Use of this guide with the Ambassador Edge Stack is not fully supported; use the more powerful, flexible ...

Reply:
Fixed